lib/document.js

```diff
@@ -10,36 +9,10 @@ export default ({ head, css, html, data, dev, staticMarkup, cdn }) => {
     </head>
     <body>
       <div id='__next' dangerouslySetInnerHTML={{ __html: html }} />
-      {staticMarkup ? null : <script dangerouslySetInnerHTML={{ __html: '__NEXT_DATA__ = ' + htmlescape(data) }} />}
-      {staticMarkup ? null : createClientScript({ dev, cdn })}
+      {staticMarkup ? null : <script dangerouslySetInnerHTML={{
+        __html: `__NEXT_DATA__ =${htmlescape(data)}; module={};`
+      }} />}
+      <script type='text/javascript' src='/main.js' />
     </body>
   </html>
 }
-
-function createClientScript ({ dev, cdn }) {
```

Comment:
I get it, that's a non issue these days as we could easily put a CDN behind all these assests. 

Specialy with on the fly CDNs.